gradlew

```diff
@@ -42,11 +42,6 @@ case "`uname`" in
     ;;
 esac
 
-# For Cygwin, ensure paths are in UNIX format before anything is touched.
-if $cygwin ; then
```

Comment:
This change was done automatically, I think due to upgrading gradle. I don't know how important it is or if it's even necessary at all. Deferring to more experienced mobile devs on that call.